Neos.Flow/Tests/Functional/ObjectManagement/Fixtures/ClassWithPhpAttributes.php

```diff
@@ -21,4 +21,12 @@
 #[SampleAttribute(ClassWithPhpAttributes::class, ['foo' => 'bar'])]
 class ClassWithPhpAttributes
 {
+    /**
+     * @return void
+     */
+    #[Flow\Around(pointcutExpression: "method(somethingImpossible())")]
```

Comment:
Now I wonder, I think we always create the "Flow" use statement in proxy classes, but what about other namespaces, do you know if we copy all use statements over to the proxy, otherwise this might fail?

Reply:
This is independent from any imports, the proxy method uses a fully qualified attribute name including the absolute namespace.

Reply:
oh I see, neat then!